app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,35 @@
+class SessionsController < ApplicationController
+  def new
```

Comment:
Why do you have an empty def)

Reply:
Because I might use it in the future :)
the guide told me :(

Reply:
Maybe use rubocop?

Reply:
It will lint all files, even the default rails files, will have to config it and im too lazy to do it now.

Reply:
You can use rubocop with rails flag :)
https://docs.rubocop.org/en/latest/cops_rails/#rails

Reply:
Will add after basic setup is done.

Reply:
You're way past basic setup ;) but okay, your repo :D

Reply:
The basic setup includes login, which is pretty basic if you ask me.

Reply:
So a 7 day project is a basic setup ;) building half of the functionality ain't basic :*

Reply:
Sorry for learning stuff, at least my tests work. Half? We have discussed all the functionality of this site in person and you know its far from half.

Reply:
Angry <3